exercises/practice/atbash-cipher/.meta/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "blurb": "Create an implementation of the atbash cipher, an ancient encryption system created in the Middle East.",
-  "authors": ["brent113"],
-  "contributors": ["axtens"],
+  "authors": ["axtens"],
+  "contributors": [],
```

Comment:
Did you mean to change the author here? I think the AtbashCipher exercise was created by axtens?

Reply:
So the atbash-cipher example originally had an encode but no decode. So I replaced the encode and wrote a matching decode. But now that I think about it, the exercise as a whole was certainly not created by me. So Iit should go back to being brent113

Reply:
With you as a contributor!